spec/traits.dd

```diff
@@ -493,6 +494,33 @@ pragma(msg, __traits(getFunctionAttributes, (int x) @trusted { return x * 2; }))
 ---
 )
 
+$(H2 $(GNAME getLinkage))
+
+	$(P Takes one argument, which is a declaration symbol, or the type of a function,
+	delegate, or pointer to function.
+	Returns a string representing the $(LINK2 attribute.html#LinkageAttribute, LinkageAttribute)
+	of the declaration.
+	The string is one of:
+	)
+
+	$(UL
+	$(LI $(D "D"))
+	$(LI $(D "C"))
+	$(LI $(D "C++"))
+	$(LI $(D "Windows"))
+	$(LI $(D "Pascal"))
+	$(LI $(D "Objective-C"))
+	$(LI $(D "System"))
+	)
+
+---
+extern (C) int fooc();
+alias aliasc = fooc;
+
+static assert(__traits(getLinkage, fooc) == "C");
+static assert(__traits(getLinkage, aliasc) == "C");
```

Comment:
I'd add tests for a few of the other linkages too (so we are sure it doesnt always return "C" ;). And also for implicit extern(D), for a pointer to function, and perhaps for a lambda and class method.